demos/gaze_estimation_demo/cpp/README.md

```diff
@@ -123,6 +123,13 @@ The following keys are supported:
 * F - to flip frames horizontally
 * Esc - to quit the demo
 
+When single image applied as an input, the demo will process and render it quickly, then exit. In this particular case, recommendation is to also apply `loop` option, which will enforce looping over processing the single image, so processed results will be continuously visualized on screen.
```

Comment:
Same here and in other readme's as well.

Reply:
done